ukis_pysat/raster.py

```diff
@@ -73,6 +73,24 @@ def arr(self):
         else:
             return reshape_as_image(self.__arr)
 
+    @arr.setter
+    def arr(self, arr_altered):
+        """array setter"""
+        if not isinstance(arr_altered, np.ndarray):
+            raise TypeError("altered array must be of type np.ndarray")
+
+        if self.dimorder == "last":

```

Comment:
@MWieland I'm not sure if we should automatically assume that the dimorder is the same and then automatically do it.

Reply:
To me it would be logic to use the dimorder of the Image instance. A user defines the dimorder upon initialization of an Image and we should not break this here. The dimorder can also be checked with `Image.dimorder` and this should match any array operations that are performed on the Image.

Reply:
Okay, but then arr_altered should just be of shape raster, reshaping without letting the user know is confusing.

Reply:
Nevermind this only happens internally, but let's describe this somewhere in the docs.